web/src/redux/utils.ts

```diff
@@ -0,0 +1,34 @@
+import {PromiseWithKnownReason} from '@reduxjs/toolkit/dist/query/core/buildMiddleware/types';
```

Comment:
I like that you moved this somewhere else, I think what I'd like to do is to give it more meaning rather than just a utils file. Maybe we can continue with the services line we are doing for the rest of the stuff.